precompiles/outposts/stride/tx_test.go

```diff
@@ -180,11 +262,14 @@ func (s *PrecompileTestSuite) TestRedeem() {
 			"success",
 			func() []interface{} {
 				return []interface{}{
-					s.keyring.GetAddr(0),
-					s.keyring.GetAddr(0),
-					common.HexToAddress(tokenPair.Erc20Address),
-					big.NewInt(1e18),
-					"stride1rhe5leyt5w0mcwd9rpp93zqn99yktsxvyaqgd0",
+					stride.AutopilotArgs{
+						ChannelID:       channelID,
+						Sender:          s.keyring.GetAddr(0),
+						Receiver:        s.keyring.GetAddr(0),
+						Token:           common.HexToAddress(tokenPair.GetErc20Address()),
```

Comment:
how come only in this occurence we're using `GetErc20Address()` instead of `Erc20Address` as in the other cases?